src/components/layouts/EvaluationLayout.tsx

```diff
@@ -341,16 +295,13 @@ function EvaluationLayout({
               </a>
             </div>
           )}
-          <VisualizationType
-            activeVisualization={
-              (showQuestionLayout && activeVisualization) ||
-              (showFeedback && CHART_TYPES.TABLE) ||
-              (showConfusionTS && CHART_TYPES.SCATTER_CHART)
-            }
-            disabled={!showQuestionLayout}
-            questionType={type}
-            onChangeType={onChangeVisualizationType}
-          />
+          {showQuestionLayout && (
```

Comment:
I just hide the chart chooser if we are on feedback or confusion views, instead of disabling it.